components/Tabs/tests/snapshot/TabBarView/MDCTabBarViewSnapshotTests.m

```diff
@@ -1212,4 +1212,82 @@ - (void)testCustomView {
   [self generateSnapshotAndVerifyForView:self.tabBarView];
 }
 
+#pragma mark - rectForItem:inCoordinateSpace:
+
+- (void)testRectForItemInCoordinateSpaceWithinBounds {
+  // Given
+  self.tabBarView.items = @[ self.item1, self.item2, self.item3 ];
+  CGSize intrinsicContentSize = self.tabBarView.intrinsicContentSize;
+  self.tabBarView.bounds =
+      CGRectMake(0, 0, intrinsicContentSize.width, intrinsicContentSize.height);
+  UIView *itemRectOverlayView = [[UIView alloc] init];
+  itemRectOverlayView.backgroundColor =
+      [UIColor.magentaColor colorWithAlphaComponent:(CGFloat)0.25];
+  [self.tabBarView addSubview:itemRectOverlayView];
+  [self.tabBarView layoutIfNeeded];
+
+  // When
+  CGRect item2FrameRect = [self.tabBarView rectForItem:self.item2
+                                     inCoordinateSpace:self.tabBarView];
+  itemRectOverlayView.frame = item2FrameRect;
+
+  // Then
+  [self generateSnapshotAndVerifyForView:self.tabBarView];
+}
+
+- (void)testRectForItemInCoordinateSpaceWhenPartiallyOutsideBarBounds {
+  // Given
+  self.tabBarView.items = @[ self.item1, self.item2 ];
+  CGSize intrinsicContentSize = self.tabBarView.intrinsicContentSize;
+  self.tabBarView.bounds = CGRectMake(0, 0, 170, intrinsicContentSize.height);
```

Comment:
suggestion: this `170` seems like a magic number. Perhaps use a constant variable with a meaningful name.

Reply:
Done.